server.js

```diff
@@ -2,65 +2,40 @@
 const express = require('express');
 const app = express();
 
-// run `npm install` to install dependencies in package.json
+const MarsMissions = require('./models/missions');
 
 // * Your mission is to complete the app
 // * The app will need routes for index and show
+
+//Index route - missions
+app.get('/missions', (req, res) => {
+  res.send(MarsMissions);
```

Comment:
you should be rendering a template here, you still need to make the template (`index.ejs`) and a folder (`views`) to put it in